src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,54 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import java.time.LocalDateTime;
+
+@Entity
+@Table(name = "moviesessions")
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne
+    private Movie movie;
+    @ManyToOne
+    private CinemaHall cinemaHall;
+    private LocalDateTime showTime;
+
+    public MovieSession() {
+    }
+
+    public MovieSession(Movie movie, CinemaHall cinemaHall) {
+        this.movie = movie;
+        this.cinemaHall = cinemaHall;
```

Comment:
The parameterized constructor does not initialize the `showTime` field, which is a required property for `MovieSession`. You should either add it as a parameter to this constructor or remove this incomplete constructor to prevent the creation of `MovieSession` instances without a `showTime`.